RecoBTag/TensorFlow/src/SecondaryVertexConverter.cc

```diff
@@ -31,7 +32,11 @@ namespace btagbtvdeep {
     sv_features.d3d = d3d_meas.value();
     sv_features.d3dsig = catch_infs_and_bound(d3d_meas.value()/d3d_meas.error(),
 					      0,-1,800);
-    sv_features.costhetasvpv = vertexDdotP(sv,pv);
+    float costhetasvpv = vertexDdotP(sv,pv);
+    if(flip){
+      costhetasvpv = -costhetasvpv;
+    }
```

Comment:
this can as well be ```sv_features.costhetasvpv = (flip ? -1.f : 1.f)* vertexDdotP(sv,pv);```

Reply:
Done here https://github.com/cms-sw/cmssw/commit/b62675266e3129ca3f77a62f2d7b29a8546d6f84#diff-dc4d48592ab74339ae57ac34bb57bd4eR35